docs/std-seq.md

```diff
@@ -12,10 +12,96 @@ the concatenation of the sequences in the array.
 | `//seq.concat(["ba", "na", "na"])` | `"banana"` |
 | `//seq.concat([[1, 2], [3, 4, 5]])` | `[1, 2, 3, 4, 5]` |
 
-## `//seq.repeat(n <: number, seq <: array) <: array` <br/> `repeat(n <: number, seq <: string) <: string`
+## `//seq.repeat(n <: number, seq <: array) <: array` <br/> `//seq.repeat(n <: number, seq <: string) <: string`
 
 `repeat` returns a sequence that contains `seq` repeated `n` times.
 
 | example | equals |
 |:-|:-|
 | `//seq.repeat(2, "hots")` | `"hotshots"` |
+
+
+## `//seq.has_prefix(prefix <: array, subject <: array) <: bool` <br/> `//seq.has_prefix(prefix <: string, subject <: string) <: bool`
+
+`has_prefix` checks whether the sequence `subject` is prefixed by sequence `prefix`. It returns a boolean.
```

Comment:
```suggestion
`has_prefix` checks whether the sequence `subject` is prefixed by sequence `prefix` and
returns true if it is, or false otherwise.
```